pkg/controller/resource-recommend/utils/k8s_resource.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright 2022 The Katalyst Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
```

Comment:
all codes in this package should be merged with `pkg/util` to make them clean and consistent

Reply:
the whole `utils` package

Reply:
Can I create a new resource-recommend directory under pkg/util to put these utils in？

Reply:
done